src/sentry/static/sentry/app/views/projectDashboard/eventList.jsx

```diff
@@ -24,6 +26,7 @@ const EventList = createReactClass({
       loading: true,
       error: false,
       statsPeriod: '24h',
+      title: this.props.type === 'new' ? t('New issues') : t('Trending issues'),
```

Comment:
Does this need to be in state?

Reply:
Not really, I just thought it was slightly nicer to evaluate once on getInitialState rather than on every render

Reply:
I think it's more correct to have it evaluate on every render since the title is a function of the prop `type`. Otherwise you could have the situation where you re-render with a different `type` and `this.state.title` doesn't get updated.